server/src/test/java/org/elasticsearch/index/shard/RemoveCorruptedShardDataCommandIT.java

```diff
@@ -555,8 +546,10 @@ public Settings onNodeStopped(String nodeName) throws Exception {
     }
 
     public void testResolvePath() throws Exception {
+        internalCluster().startMasterOnlyNode();
```

Comment:
I think this one can be fixed by moving the scope to ESIntegTestCase.Scope.TEST. As these tests are starting up nodes anyway, there's no need to keep the cluster per suite. Maybe that also fixes the testCorruptTranslogTruncationOfReplica

Reply:
https://github.com/elastic/elasticsearch/pull/36367/commits/4d9d141b90c390068f1374a471309e7e8d7cf006 this helps this test but does not help another test in this class (it has different root cause)